src/Bicep.Core.Samples/Files/InvalidExpressions_LF/main.diagnostics.bicep

```diff
@@ -149,7 +149,7 @@ var ternary = null ? 4 : false
 // complex expressions
 var complex = test(2 + 3*4, true || false && null)
 //@[4:11) [BCP028 (Error)] Identifier "complex" is declared multiple times. Remove or rename the duplicates. |complex|
-//@[14:18) [BCP082 (Error)] The name "test" does not exist in the current context. Did you mean "test1"? |test|
+//@[14:18) [BCP057 (Error)] The name "test" does not exist in the current context. |test|
```

Comment:
Spelling suggestions seem to have regressed here?

Reply:
Actually I'm not sure about this one - in some ways it seems like an improvement. `test` is being used as a function, and `test1` is not a function - it's an identifier which has been declared in the file.

Reply:
Ahh good point. Do we need to add an equivalent test since we're losing this one?

Reply:
We've got other tests in this file for BCP082, so it should be fine.

Reply:
Sounds good.